src/debug.h

```diff
@@ -48,7 +50,7 @@ void _on_assert_fail(const char* vara, const char* op, const char* varb,
   }
```

Comment:
Have you considered including the stack trace information in the exception message?